### PR TITLE
chore: Added update grub to fix bootstrap

### PR DESCRIPTION
**Pull request by jamtur01**
Signed-off-by: James Turnbull <james@lovedthanlost.net>

### CODE REVIEW

**Comment by jamtur01** [comment]
In reference to https://github.com/actions/virtual-environments/issues/1605.

File: scripts/environment/bootstrap-ubuntu-20.04.sh [hunk 1]
```diff
@@ -3,7 +3,11 @@ set -e -o verbose
 
 export DEBIAN_FRONTEND=noninteractive
 
+apt-get install -y grub-efi
```
Comment (jszwedko) [inline scripts/environment/bootstrap-ubuntu-20.04.sh:6]: Not a big deal in this particular case, but I generally prefer to use the longer form of flags, that is `--yes`, in scripts to make them easier to read without pulling up the man page.

File: scripts/environment/bootstrap-ubuntu-20.04.sh [hunk 2]
```diff
@@ -3,7 +3,11 @@ set -e -o verbose
...
+update-grub
+
 apt update -y -qq
+apt upgrade -y -qq
```
Comment (jszwedko) [inline scripts/environment/bootstrap-ubuntu-20.04.sh:10]: Did you end up needing this? I see it wasn't mentioned in https://github.com/actions/virtual-environments/issues/1605

Comment (jamtur01) [reply scripts/environment/bootstrap-ubuntu-20.04.sh:10]: Yeah - that's the original step that was failing. The grub-efi packages was dying during the apt upgrade.

**Review by jszwedko — APPROVED** [review]

**Pull request merged**